server/routes/volunteers.js

```diff
@@ -24,11 +26,14 @@ router.get('/emailsignup', (req, res) => {
     })
     .catch(err => {
       log(err.message)
-      res.redirect(`./email-volunteer-error/${volunteer.userId}/${volunteer.eventId}`)
+      res.status(500).json({
+        error: {
+          title: 'Unable to register from email'
+        }
+      })
     })
 })
 
-// include getTokenDecoder() like function into post route that passes authorisation header?REQUIRES TOKEN
 // Verifies the data being modified belongs to the user that added it. --------------------
```

Comment:
I'm not sure why someone put the dashes in these original comments, but feel free to remove them.